NullPointerException in HTMLReportGenerator

### Summary of the Bug
The `HTMLReportGenerator` throws a NPE during the generation of the HTML report.
Stacktrace:
```
Caused by: java.lang.NullPointerException

        at org.apache.commons.io.FileUtils.copyURLToFile(FileUtils.java:1456)

        at org.eclipse.sw360.antenna.workflow.generators.HTMLReportGenerator.produce(HTMLReportGenerator.java:67)

        at org.eclipse.sw360.antenna.api.workflow.AbstractGenerator.produce(AbstractGenerator.java:22)

        at org.eclipse.sw360.antenna.workflow.AntennaWorkflow.generateOutputViaGenerators(AntennaWorkflow.java:130)

        at org.eclipse.sw360.antenna.workflow.AntennaWorkflow.execute(AntennaWorkflow.java:57)

        at org.eclipse.sw360.antenna.core.AntennaCore.compose(AntennaCore.java:51)

        at org.eclipse.sw360.antenna.frontend.stub.cli.AbstractAntennaCLIFrontend.execute(AbstractAntennaCLIFrontend.java:94)

        at org.eclipse.sw360.antenna.frontend.stub.gradle.AntennaImpl.execute(AntennaImpl.java:52)

        at org.eclipse.sw360.antenna.frontend.stub.gradle.AnalyzeTask.analyze(AnalyzeTask.java:42)

        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:103)

        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:49)

        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:42)

        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:28)

        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:717)

        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:684)

        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$5.run(ExecuteActionsTaskExecuter.java:476)

        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)

        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)

        at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
```

### Steps to reproduce
- Build `example-project`

### Acceptance Criteria
- [x] The `HTMLReportGenerator` works properly again.

### Definition of Done
- Acceptance criteria fulfilled
- A test case is created to reproduce the bug
- A PR is created, the CI infrastructure reports green, the bug test case proves that bug is fixed
- The PR is reviewed and approved
- No TODOs left in the code unless explained in the ticket, if something else is still open, this is summarized in a comment in the issue
- Documentation is updated

--- a/core/core-workflow-steps/src/main/java/org/eclipse/sw360/antenna/workflow/generators/HTMLReportGenerator.java
+++ b/core/core-workflow-steps/src/main/java/org/eclipse/sw360/antenna/workflow/generators/HTMLReportGenerator.java
@@ -57,7 +57,7 @@ public class HTMLReportGenerator extends AbstractGenerator {
         // Get path to disclosure documents
         ToolConfiguration toolConfig = context.getToolConfiguration();
         Path reportFilePath = toolConfig.getAntennaTargetDirectory().resolve(LICENSE_REPORT_FILE);
-        URL styleSource = getClass().getResource(File.separator + LICENSE_REPORT_STYLE_FILE);
+        URL styleSource = getClass().getClassLoader().getResource(LICENSE_REPORT_STYLE_FILE);
         Path reportStyleFilePath = toolConfig.getAntennaTargetDirectory().resolve(LICENSE_REPORT_STYLE_FILE);
 
         Set<ArtifactForHTMLReport> artifactsForHTMLReport = extractRelevantArtifactInformation(artifacts);

